### DISCUSSION TITLE
Added data on the map is not being saved

### DISCUSSION THREAD

**Discussion by floriansenf** [opengisch/QField#4961] [Q&A]
Hello,

I tried to add points and other data to the map. But other users don't see the points and when I close the app and restart it they are also gone.

I pushed changes. In the admin area in Log entries I see that data has been edited/added. But other users don't see it and when I synchronize the project my added data is gone and I also don't see data of other users.

Interestingly I uninstalled the app and installed it again. Then I see data of other users. But as soon as other users add new data and I synchronize the project I have the issue again and I don't see their data and they also don't see my added data.

Did anyone had this issue already? How to solve this?

**Comment by SeqLaz** [comment]
Hello @floriansenf ,

Sometimes issues arise when changes in the schema of the GeoPackage are made. To resolve this, we recommend exporting your layer as a copy. Right-click on the layer and select "Save as...". After exporting, you can copy all the styles from the original layer and paste them onto the new layer.
